resources/views/livewire/custom-field-set-default-values-for-model.blade.php

```diff
@@ -1,54 +1,79 @@
+                {{ Form::checkbox('add_default_values', 1, old('add_default_values', $add_default_values), ['data-livewire-component' => $_instance->id, 'id' => 'add_default_values', 'wire:model' => 'add_default_values']) }}
+                {{ trans('admin/models/general.add_default_values') }}
+            </label>
+        </div>
     </div>
+
     @if ($this->add_default_values ) {{-- 'if the checkbox is enabled *AND* there are more than 0 fields in the fieldsset' --}}
-    <div style="padding-left: 10px; padding-bottom: 0px; margin-bottom: -15px;">
-        <div class="form-group">
             @if ($fields)
+
                 @foreach ($fields as $field)
                     <div class="form-group">
-                    
-                            <label class="col-md-3 control-label{{ $errors->has($field->name) ? ' has-error' : '' }}" for="default-value{{ $field->id }}">{{ $field->name }}</label>
 
-                            <div class="col-md-7">
+                        <label class="col-md-3 control-label{{ $errors->has($field->name) ? ' has-error' : '' }}">{{ $field->name }}</label>
+
+                        <div class="col-md-7">
+
                                 @if ($field->format == "DATE")
+
                                     <div class="input-group col-md-4" style="padding-left: 0px;">
                                         <div class="input-group date" data-provide="datepicker" data-date-format="yyyy-mm-dd"  data-autoclose="true">
                                             <input type="text" class="form-control" placeholder="{{ trans('general.select_date') }}" name="default_values[{{ $field->id }}]" id="default-value{{ $field->id }}" value="{{ $field->defaultValue($model_id) }}">
                                             <span class="input-group-addon"><i class="fas fa-calendar" aria-hidden="true"></i></span>
                                         </div>
                                     </div>
+
                                 @elseif ($field->element == "text")
-                                    <input class="form-control m-b-xs" type="text" value="{{ $field->defaultValue($model_id) }}" id="default-value{{ $field->id }}" name="default_values[{{ $field->id }}]">
+
+
+                                        <input class="form-control" type="text" value="{{ $field->defaultValue($model_id) }}" id="default-value{{ $field->id }}" name="default_values[{{ $field->id }}]">
+
+
                                 @elseif($field->element == "textarea")
-                                    <textarea class="form-control" style="width: 100%;" id="default-value{{ $field->id }}" name="default_values[{{ $field->id }}]">{{ $field->defaultValue($model_id) }}</textarea><br>
+
+
+                                        <textarea class="form-control" style="width: 100%;" id="default-value{{ $field->id }}" name="default_values[{{ $field->id }}]">{{ $field->defaultValue($model_id) }}</textarea>
```

Comment:
I know these types of spacing changes are always nice to have, but they really do make things harder to review, especially when there are subtle changes that I might not be able to 'eyeball'. As always, not a hill I'm gonna die on, but would prefer these not be in here.

Reply:
I know, but it makes it harder to read longer term if they don't get done now.